build.rs

```diff
@@ -38,11 +38,24 @@ fn main() {
         });
     }
 
-    env::set_current_dir(boehm_src).unwrap();
+    env::set_current_dir(&boehm_src).unwrap();
 
     run("./autogen.sh", |cmd| cmd);
-    run("./configure", |cmd| cmd.env("CFLAGS", POINTER_MASK));
+    run("./configure", |cmd| {
+        cmd.arg("--enable-static")
+            .arg("--disable-shared")
+            .env("CFLAGS", POINTER_MASK)
+    });
 
     let cpus = num_cpus::get();
     run("make", |cmd| cmd.arg("-j").arg(format!("{}", cpus)));
+
+    let mut libpath = PathBuf::from(&boehm_src);
+    libpath.push(".libs/");
```

Comment:
We don't need the trailing `/` I think?

Reply:
Probably not, but sometimes I leave it in as it shows that you intend for the path to be a directory. I don't feel strongly.

Reply:
Good catch. This shouldn't really be hard-coded either, try this: 724bda7260cf2e3864a78ffa16f23f1425c55466